modules/apps/object/object-test/src/testFunctional/tests/ObjectFieldsDateAndTime.testcase

```diff
@@ -64,5 +64,90 @@ definition {
 		}
 	}
 
+	@description = "LPS-143068  - Verify if is possible to see DateTime with differents Time Zone"
+	@priority = 5
+	test CanSeeDifferentTimeZone {
+		property portal.acceptance = "true";
+
+		task ("Given a custom object with a date and time field as UTC") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 186248",
+				objectName = "CustomObject186248",
+				pluralLabelName = "Custom Objects 186248");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject186248");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 186248");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Custom Date Time",
+				fieldTimeStorage = "Convert to UTC",
+				fieldType = "Date and Time");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject186248");
+
+			User.openUsersAdmin();
+
+			UserNavigator.gotoUser(userScreenName = "test");
+
+			UserNavigator.gotoDisplaySettings();
+
+			Select(
+				locator1 = "Select#TIME_ZONE",
+				value1 = "(UTC) Coordinated Universal Time");
+
+			PortletEntry.save();
```

Comment:
I'm not sure what this is accomplishing in terms of the test, can you explain why were adding these steps?